tests/components/google_drive/test_backup.py

```diff
@@ -47,6 +47,7 @@
     "backup_id": "test-backup",
     "database_included": True,
     "date": "2025-01-01T01:23:45.678Z",
+    "extra_metadata": {},
```

Comment:
Do the tests pass? Don't you need to add `"with_automatic_settings": False`?
Should we replace the constant with:
TEST_AGENT_BACKUP_RESULT = TEST_AGENT_BACKUP.as_dict() | {...}

Reply:
Yes, the tests pass.
Yes, it could be a good idea to decrease the verbosity of the tests by doing some trick like that, but that's for a separate PR.

Reply:
Tests are failing. 😉